packages/react/src/components/dateInput/DateInput.tsx

```diff
@@ -55,9 +55,25 @@ export type DateInputProps = Omit<TextInputProps, 'onChange'> & {
    */
   ref?: React.Ref<HTMLInputElement>;
   /**
-   * Disables date(s) based on conditional function
+   * Disables date(s) based on conditional function.
    */
   isDateDisabledBy?: (date) => boolean;
+  /**
+   * Set custom theme for specific dates.
+   */
+  setDateTheme?: (date: Date) => DateCustomTheme | undefined;
+  /**
+   * Legend items for the date picker.
+   * @example
+   * ```ts
+   * legend=[{ elementId: 'legend-item-1', label: 'Text label', color: #c3c3c3 }, { elementId: 'legend-item-2', label: 'Text label', color: #d6d66d }}]
+   * ```
```

Comment:
Nice, you can add ts examples here! Should the value of the `color` also be quoted like other props? Plus there's double `}} `in the end, but only one curly brace is open. Too bad linter can't see these.

Reply:
Nice find, fixed!